rotkehlchen/errors.py

```diff
@@ -123,3 +123,11 @@ class ModuleInitializationFailure(Exception):
 
 class ModuleInactive(Exception):
     """Raises when call happens for a module that is not initialized"""
+
+
+class WriteError(Exception):
```

Comment:
There is no reason to create a new exception for something like this. Simply catch the OS.Error